lnrpc/walletrpc/walletkit.proto

```diff
@@ -768,6 +768,9 @@ message FundPsbtRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 7;
+
+    // Input timelock in seconds.
```

Comment:
To avoid confusion about what this parameter actually does (so it isn't confused with other locking mechanisms such as CLTV/CSV that are used in channel related transactions), this should probably be called `utxo_lease_expiration_seconds` and the comment should also be more detailed.
For example (also see `walletkit.proto`, `LeaseOutputRequest`):
```
    // The time in seconds before the wallet's lock on the selected UTXOs expires. If set to zero, the default
    // lock duration is used.
```